src/test/java/org/extism/chicory/sdk/HostEnvTest.java

```diff
@@ -0,0 +1,28 @@
+package org.extism.chicory.sdk;
+
+import com.dylibso.chicory.log.SystemLogger;
+import junit.framework.TestCase;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+public class HostEnvTest extends TestCase {
+    public void testShowcase() {
+        var logger = new SystemLogger();
+
+        var config = Map.of("key", "value");
+        var hostEnv = new HostEnv(new Kernel(logger), config, logger);
+
+        assertEquals(hostEnv.config().get("key"), "value");
+
+        byte[] bytes = "extism".getBytes(StandardCharsets.UTF_8);
+        hostEnv.var().set("test", bytes);
+        assertSame(hostEnv.var().get("test"), bytes);
+
+        hostEnv.log().log(LogLevel.INFO, "hello world");
```

Comment:
Just a side note, should we consider convenience methods for the levels? `hostEnv.log().info("hello world");`